libs/langchain-google-cloud-sql-pg/src/engine.ts

```diff
@@ -10,6 +10,44 @@ export interface PostgresEngineArgs {
   iamAccountEmail?: string,
 }
 
+export interface VectorStoreTableArgs {
+  tableName: string,
+  vectorSize: number,
```

Comment:
Similar to the Engine initialization, can we have the tableName and vectorSize as required args and separate from the Args dict of optional ars?